modules/cert-manager-configure-cloud-credentials-gcp-non-sts.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-non-sts-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-prepare-cloud-credentials-gcp-non-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} on GCP
+
+To configure the cloud credentials for the {cert-manager-operator} on a GCP cluster you must create a `CredentialsRequest` object and allow the Cloud Credential Operator to generate the cloud credentials secret.
+
+.Prerequisites
+
+* The {cert-manager-operator} 1.11.1 or later is installed.
+
+.Procedure
+
+. Create a `CredentialsRequest` resource YAML file, for example, `sample-credential-request.yaml`, as follows:
++
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  generation: 1
+  labels:
+    app: cert-manager
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/instance: cert-manager
+    app.kubernetes.io/name: cert-manager
+  name: cert-manager
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: GCPProviderSpec
+    predefinedRoles:
+    - roles/dns.admin
+  secretRef:
+    name: gcp-credentials
+    namespace: cert-manager
+  serviceAccountNames:
+  - cert-manager
+----
+
+. Create a `CredentialsRequest` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sample-credential-request.yaml
+----
+
+. Apply the secrets generated in the manifests directory of your cluster by running the following command:
```

Comment:
Missed it in earlier reviews, this step is also not required and we should remove it.